src/typescript/sdk/tests/e2e/arena/general/arena.test.ts

```diff
@@ -497,3 +553,179 @@ describe("ensures leaderboard history is working", () => {
+    );
+    await waitForProcessor(res);
+
+    const arenaInfo = await fetchArenaInfo({});
+
+    expect(arenaInfo).toBeTruthy();
+    expect(arenaInfo!.meleeID).toEqual(melee.view.meleeID);
+    expect(arenaInfo!.duration).toEqual(melee.view.duration);
+    expect(arenaInfo!.startTime).toEqual(melee.view.startTime);
+    expect(arenaInfo!.maxMatchAmount).toEqual(melee.view.maxMatchAmount);
+    expect(arenaInfo!.maxMatchPercentage).toEqual(melee.view.maxMatchPercentage);
+    expect(arenaInfo!.volume).toEqual(volume);
+    expect(arenaInfo!.emojicoin0Locked).toEqual(
+      account3EnterRes.events.arenaEnterEvents[0].emojicoin0Proceeds
+    );
+    expect(arenaInfo!.emojicoin1Locked).toEqual(0n);
+  }, 30000);
+
+  it("verifies that arena info data is correct 2", async () => {
+    let volume = 0n;
+    let emojicoin0Locked = 0n;
+    let emojicoin1Locked = 0n;
+
+    const accounts = [
+      getFundedAccount("420"),
+      getFundedAccount("421"),
+      getFundedAccount("422"),
+      getFundedAccount("423"),
+      getFundedAccount("424"),
+      getFundedAccount("425"),
+      getFundedAccount("426"),
+      getFundedAccount("427"),
+    ];
+
+    for (const account of accounts.map((a, i) => [a, i] as [Ed25519Account, number])) {
+      const enterRes = await enterHelper(
+        account[0],
+        `symbol${(account[1] % 2) + 1}` as "symbol1" | "symbol2"
+      );
+      volume += enterRes.events.arenaEnterEvents[0].quoteVolume;
+      emojicoin0Locked += enterRes.events.arenaEnterEvents[0].emojicoin0Proceeds;
+      emojicoin1Locked += enterRes.events.arenaEnterEvents[0].emojicoin1Proceeds;
+    }
+
+    for (const account of accounts) {
+      const swapRes = await emojicoin.arena.swap(
+        account,
+        melee.market1.symbolEmojis,
+        melee.market2.symbolEmojis
+      );
+      volume += swapRes.events.arenaSwapEvents[0].quoteVolume;
+      const diff = getEmojicoinLockedDiffFromSwapRes(swapRes, melee);
+      emojicoin0Locked += diff.emojicoin0Locked;
+      emojicoin1Locked += diff.emojicoin1Locked;
+    }
+
+    for (const account of accounts.slice(0, Math.floor(accounts.length / 2))) {
+      const swapRes = await emojicoin.arena.swap(
+        account,
+        melee.market1.symbolEmojis,
+        melee.market2.symbolEmojis
+      );
+      volume += swapRes.events.arenaSwapEvents[0].quoteVolume;
+      const diff = getEmojicoinLockedDiffFromSwapRes(swapRes, melee);
+      emojicoin0Locked += diff.emojicoin0Locked;
+      emojicoin1Locked += diff.emojicoin1Locked;
+    }
+
+    for (const account of accounts.slice(Math.floor(accounts.length / 2))) {
+      const exitRes = await emojicoin.arena.exit(
+        account,
+        melee.market1.symbolEmojis,
+        melee.market2.symbolEmojis
+      );
+      emojicoin0Locked -= exitRes.events.arenaExitEvents[0].emojicoin0Proceeds;
+      emojicoin1Locked -= exitRes.events.arenaExitEvents[0].emojicoin1Proceeds;
+    }
+
+    await sleep(1000);
```

Comment:
```suggestion
    // Wait for the indexer to process the transactions.
    await sleep(1000);
```

(or whatever this is for- I assume it's to wait on the indexer)